Packs/FeedRSS/Integrations/FeedRSS/FeedRSS.py

```diff
@@ -0,0 +1,146 @@
+        self.content_max_size = content_max_size * 1000
+        self.parsed_indicators = []
+        self.feed_data = None
+        self.feed_response = self.request_feed_url()
+
+    def request_feed_url(self):
+        return self._http_request(method='GET', resp_type='response')
+
+    def create_indicators_from_response(self):
+        parsed_indicators: list = []
+        if not self.feed_data:
+            raise DemistoException(f"Could not parse feed data {self._base_url}")
+
+        for indicator in reversed(self.feed_data.entries):
+            publications = []
+            if indicator:
+                published = email.utils.parsedate(indicator.published)
+                if published:
+                    published_iso = datetime.fromtimestamp(mktime(published)).isoformat()
+                    publications.append({
+                        'timestamp': indicator.get('published'),
+                        'link': indicator.get('link'),
+                        'source': self._base_url,
+                        'title': indicator.get('title')
+                    })
+                    text = self.get_url_content(indicator.get('link'))
+                    indicator_obj = {
+                        "type": 'Report',
+                        "value": indicator.get('title').replace(',', ''),  # Remove comma because the script of create
+                        # relationship includes that as a list of titles.
+                        "rawJSON": {'value': indicator, 'type': 'Report', "firstseenbysource": published_iso},
+                        "fields": {
+                            'rawcontent': text,
+                            'publications': publications,
+                            'description': indicator.get('summary'),
+                            'tags': self.feed_tags,
+                        }
+                    }
+                    if self.tlp_color:
+                        indicator_obj['fields']['trafficlightprotocol'] = self.tlp_color
+
+                parsed_indicators.append(indicator_obj)
+
+        return parsed_indicators
+
+    def get_url_content(self, link: str):
+        """Returns the link content only from the relevant tags (listed on HTML_TAGS). For better performance - if the
+         extracted content is bigger than "content_max_size" we trim him"""
+
+        response_url = self._http_request(method='GET', full_url=link, resp_type='str')
+        report_content = 'This is a dumped content of the article. Use the link under Publications field to read ' \
+                         'the full article. \n\n'
+        soup = BeautifulSoup(response_url.text, "html.parser")
+        for tag in soup.find_all():
+            if tag.name in HTML_TAGS:
+                for string in tag.stripped_strings:
+                    report_content += ' ' + string
+        if len(report_content.encode('utf-8')) > self.content_max_size:  # Ensure report_content does not exceed the
```

Comment:
let's encode once and to a var, and do a fallback in case encoding fails. @yaakovi recommendations?

Reply:
I read the documentation of encode(), I changed the code to do: `encoded_content = report_content.encode('utf-8', errors='replace')` which replace the character that can not be encoded with a questionmark. Sounds ok?

Reply:
yes